README.md

```diff
@@ -1,50 +1,81 @@
 # Making Tax Digital for Income Tax API Changelog
 
-This page contains a log of the latest changes for [Making Tax Digital for Income Tax APIs](https://developer.service.hmrc.gov.uk/api-documentation/docs/api?filter=income-tax-mtd). For information about planned changes to these APIs, see the [Making Tax Digital for Income Tax roadmap](https://developer.service.hmrc.gov.uk/roadmaps/mtd-itsa-vendors-roadmap/).
+This page contains a log of the latest changes
```

Comment:
Can you please undo all the formatting changes you've made in this file? There might be something wrong with your scala formatter.